HyperV/scripts/export-eventlog.ps1

```diff
@@ -2,9 +2,11 @@
 
 . "C:\OpenStack\nova-ci\HyperV\scripts\config.ps1"
 
+
+
 function dumpeventlog($path){
 	
-	Get-Eventlog -list | ForEach-Object {
+	Get-Eventlog -list | Where-Object { $_.Entries -ne '0' } | ForEach-Object {
 		$logFileName = $_.LogDisplayName
 		$exportFileName =$path + "\eventlog_" + $logFileName + (get-date -f yyyyMMdd) + ".evt"
```

Comment:
remove date from name please, we don't need it since we know it's for the current build.